packages/stargate/src/index.ts

```diff
@@ -1,3 +1,7 @@
+/*
+ * This codec is derived from the Cosmos SDK protocol buffer definitions and can change at any time.
+ */
```

Comment:
Any chance we can turn this into a doc domment (starting with `/**` or `///`)? I don't know if this can be done for re-rexported symbols.